src/components/Body/MovieSettings/MovieSettings.tsx

```diff
@@ -1,6 +1,6 @@
 import { useState } from "react";
 
-import DeleteModal from "../../Modal/DeleteModal";
+import DeleteModal from "../../Modal/DeleteModal/DeleteModal";
```

Comment:
I didn't notice this last time, you need to make sure that when you open the menu on the card, `Movie details` does not open
![image](https://user-images.githubusercontent.com/106384184/210488634-9372a404-a973-433b-baf5-d33c059dcf5f.png)

Hint: think about event phases